action.yml

```diff
@@ -0,0 +1,35 @@
+name: Godot toolkit install
+author: Pawel Lampe
+description: Install the Godot toolkit
+branding:
+  icon: activity
+  color: blue
+
+inputs:
+  version:
+    description: Version of the gdtoolkit to install
+    default: "4.*"
+
+runs:
+  using: composite
+  steps:
+    - id: create-requirements-if-not-exists
+      run: |
+        if [ ! -f requirements.txt ]; then
+          echo "gdtoolkit==${{inputs.version}}" > requirements.txt
+        fi
```

Comment:
This is only needed, because otherwise the `actions/setup-python@v4` with `cache: "pip"` would fail
It searches for a `requirements.txt` or `pyproject.toml`
If this does not exists, the `install-python` would fail :shrug: 

Because Godot projects are usually not a python project, we need to fake that file